app/_src/gateway/licenses/index.md

```diff
@@ -17,10 +17,18 @@ functionality, {{site.base_gateway}} enforces the presence and validity of a
 
 ## Deploying the license file
 
+{% if_version lte:3.4.x %}
```

Comment:
@ms2008 Hi! I added conditional rendering because I wasn't sure if licenses would behave the old way on older versions of Kong. If the new license behavior applies to all Kong versions, I can remove the conditional rendering.

Reply:
Regarding this section of the license file, please remove the conditional rendering, it applies to all Kong versions.